pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -121,6 +121,25 @@ func (b *BlackBoxExporter) EnsureBlackBoxExporterServiceExists() error {
 	return nil
 }
 
+func (b *BlackBoxExporter) EnsureBlackBoxExporterConfigMapExists() error {
+	resource := corev1.ConfigMap{}
+	populationFunc := func() corev1.ConfigMap { return templateForBlackBoxExporterConfigMap(b.NamespacedName) }
```

Comment:
commenting on this function is verbose, but matches the style of the code in this file